docs/CHANGELOG

```diff
@@ -0,0 +1,125 @@
+v. 8.0.0-beta
+~~~~~~~~~~~~~
+
+Backend Support for Public and Private Clouds
+=============================================
+
+Zenko supports multiple-cloud storage operations natively using the Amazon
+Simple Storage Service (S3) protocol. Once a site has been integrated to
+the S3 protocol, Zenko handles S3 API calls and can replicate stored data at
+the site level to the following supported private and public clouds: 
+
+-  Amazon S3-based public clouds:
+
+   -  Amazon Simple Storage Service
+   -  Digital Ocean Spaces
+   -  Wasabi
+
+-  Amazon-S3-based private clouds:
+
+   - Scality RING S3 Connector
+
+-  Other public clouds:
+
+   -  Google Cloud Storage
+   -  Microsoft Azure Storage
+
+Other cloud storage services are being validated.
+
+CRR Retry
+=========
+
+As with any large data transfers over public infrastructure,
+cross-region replication operations are imperfect and can suffer
+failure. In early releases, CRR operations did not fail gracefully--sometimes
+requiring repetition of an entire CRR operation. With v. 1.0, Zenko provides
+a way to monitor CRR operations, retrieve a list of failed operations,
+and retry specific operations on failure.
+
+If a CRR request fails, CRR Retry provides a list of all failed CRR
+requests, formatted similarly to S3’s List Object API response. Users can
+select failed CRR requests to retry by specifying the bucket, key,
+version ID and storage class for each listed CRR request failure.
+
+CRR Retry is accessible using a RESTful API call to the Backbeat
+component.
+
+S3 Cross-Region Replication (CRR) Statistics
+============================================
+
+This feature enables replication status tracking. The replication system
+exposes metrics through a REST API to monitor the replication of
+pending, processing, and completed objects. It returns the number of
+errors that occurred during replication, the current throughput—in

```

Comment:
I'm not sure if this is provided by the promclient metrics, but the kafka metrics we collect currently don't record errors